src/servers/src/http.rs

```diff
@@ -587,6 +587,16 @@ impl HttpServerBuilder {
         }
     }
 
+    pub fn with_log_ingest_handler(self, handler: LogHandlerRef) -> Self {
+        Self {
+            router: self.router.nest(
+                &format!("/{HTTP_API_VERSION}/event"),
```

Comment:
Should we discuss the HTTP prefix here, in case we need to add some more routes later for pipeline model CRUD
cc @yuanbohan